src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -982,7 +982,7 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
 
         $chainLoader->replaceArgument(0, $serializerLoaders);
 
-        if (isset($config['cache']) && $config['cache']) {
+        if (!$container->getParameter('kernel.debug')) {
```

Comment:
you can, you just have to `--env=prod`
since there is no invalidation, enabling this cache in dev only creates WTFs :-)
We did a similar change for validation:
https://github.com/symfony/symfony/pull/18544/files#diff-0e793081ceb720201745c982a568903fL784